### DISCUSSION TITLE
Fusion

### DISCUSSION THREAD

**Discussion by JosipNizetic** [AxisCommunications/discussions#218] [VAPIX] [answered]
Hello,

I am currently doing integrations with AXIS Q1656-DLE Radar-Video Fusion Camera.
I followed your guidelines with  **/axis-cgi/featureflag.cgi** and have enabled **radar_video_fusion_metadata**. That is where I saw following property 
```
            {
                "name": "best_snapshot",
                "value": false,
                "description": "Run best snapshot in metadata fusion.",
                "defaultValue": false
            },
```
Could you please explain me what this parameter do? Or point me where can i find explanation. 
My global goal is to find out if there is possibility to get snap-shot from Event or Anyltic data? Or I need to grab frame form video stream. 

Thank you very much,
Josip

**Comment by vivekatoffice** [comment]
Related link:
[Feature Flag Service](https://www.axis.com/vapix-library/subjects/t10175981/section/t10177668/display?section=t10177668-t10177666)
![image](https://github.com/AxisCommunications/discussions/assets/52032125/639002e1-6a9b-48ba-9cec-0d0e537c8589)

**Comment by JosipNizetic** [comment]
I know how to read documentation so this is totally useless.

**Reply by vivekatoffice** [reply]
Sorry for the trouble. I will try to explore more on this and come back if I find more details. I am very new to the " /axis-cgi/featureflag.cgi" API 😉. More to learn from experts like you.

*If your question remains unanswered for a period of time, please revisit it to see whether it can be improved by following the guidelines listed in [Axis support guidelines](https://github.com/orgs/AxisCommunications/discussions/13).*

**Comment by JosipNizetic** [comment]
No problem really. I am not export myself that is reason I ask questions here :-). I meant no harm.
If that is feature flag that is obsolete (  feature wen into regular firmware) that is also answer...

**Comment by vivekatoffice** [comment, marked as answer]
Hi @JosipNizetic ,
The [AXIS Scene Metadata integration](https://www.axis.com/developer-community/scene-metadata-integration) guide is available now. 

![image](https://github.com/AxisCommunications/discussions/assets/52032125/4983f6d3-2a8d-4bba-ba9d-c0e7ac94145a)
